x-pack/solutions/observability/plugins/slo/public/pages/slo_management/components/slo_management_table.tsx

```diff
@@ -0,0 +1,358 @@
+      name: (slo: SLODefinitionResponse) =>
+        slo.enabled
+          ? i18n.translate('xpack.slo.item.actions.disable', {
+              defaultMessage: 'Disable',
+            })
+          : i18n.translate('xpack.slo.item.actions.enable', {
+              defaultMessage: 'Enable',
+            }),
+      description: (slo: SLODefinitionResponse) =>
+        slo.enabled
+          ? i18n.translate('xpack.slo.item.actions.disable', {
+              defaultMessage: 'Disable',
+            })
+          : i18n.translate('xpack.slo.item.actions.enable', {
+              defaultMessage: 'Enable',
+            }),
+      'data-test-subj': 'sloActionsManage',
+      enabled: () => !!permissions?.hasAllWriteRequested,
+      onClick: (slo: SLODefinitionResponse) => {
+        const isEnabled = slo.enabled;
+
+        if (isEnabled) {
+          setSloToDisable(slo);
+        } else {
+          setSloToEnable(slo);
+        }
+      },
+    },
+    {
+      type: 'icon',
+      icon: 'trash',
+      name: i18n.translate('xpack.slo.item.actions.delete', {
+        defaultMessage: 'Delete',
+      }),
+      description: i18n.translate('xpack.slo.item.actions.delete', {
+        defaultMessage: 'Delete',
+      }),
+      'data-test-subj': 'sloActionsDelete',
+      enabled: (slo: SLODefinitionResponse) => !!permissions?.hasAllWriteRequested,
+      onClick: (slo: SLODefinitionResponse) => {
+        setSloToDelete(slo);
+      },
+    },
+
+    {
+      type: 'icon',
+      icon: 'refresh',
+      name: i18n.translate('xpack.slo.item.actions.reset', {
+        defaultMessage: 'Reset',
+      }),
+      description: i18n.translate('xpack.slo.item.actions.reset', {
+        defaultMessage: 'Reset',
+      }),
+      'data-test-subj': 'sloActionsReset',
+      enabled: () => !!permissions?.hasAllWriteRequested,
+      onClick: (slo: SLODefinitionResponse) => {
+        setSloToReset(slo);
+      },
+    },
+  ];
+
+  const columns: Array<EuiBasicTableColumn<SLODefinitionResponse>> = [
+    {
+      field: 'name',
+      name: i18n.translate('xpack.slo.sloManagementTable.columns.nameLabel', {
+        defaultMessage: 'Name',
+      }),
+      render: (_, slo: SLODefinitionResponse) => {
+        return (
+          <EuiLink
+            data-test-subj="sloDetailsLink"
+            href={http.basePath.prepend(sloPaths.sloDetails(slo.id, ALL_VALUE))}
+            target="_blank"
+          >
+            {slo.name}
+          </EuiLink>
+        );
+      },
+    },
+    {
+      field: 'version',
+      width: '10%',
+      name: i18n.translate('xpack.slo.sloManagementTable.columns.versionLabel', {
+        defaultMessage: 'Version',
+      }),
+      render: (item: SLODefinitionResponse['version']) => {
+        return item < SLO_MODEL_VERSION ? (
+          <EuiFlexGroup alignItems="center" direction="row" gutterSize="xs">
+            <EuiFlexItem grow={0}>
+              <EuiText size="s">{item}</EuiText>
```

Comment:
I think we should translate the raw version to something that makes more sense for the user, e.g. Outdated or Up-to-date